src/app/custom-decorator/method-decorators/print-with-parameter.decorator.ts

```diff
@@ -0,0 +1,13 @@
+export function printWithParameter(message: string): (target: Object, propertyName: string, descriptor: PropertyDescriptor) => PropertyDescriptor {
+  return function (target: Object, propertyName: string, descriptor: PropertyDescriptor): PropertyDescriptor {
+    const originalMethod = descriptor.value;
+
+    descriptor.value = function (...args: any[]): PropertyDescriptor {
```

Comment:
we can use arrow functions, function expression is not mostly used in JS nowadays

Reply:
it's about all files in this PR